docs/compute/python.md

```diff
@@ -1 +1,151 @@
-# Python docs
\ No newline at end of file
+# Python environment
+
+Hopsworks provides a premade environment for data engineering, machine learning and more general data science development.
```

Comment:
```suggestion
Hopsworks comes out of the box with a Python environment for data engineering, machine learning and more general data science development.
```

It's fine to keep the current text, just thought of a suggestion to make the text more context-specific